app/Http/Controllers/NoteController.php

```diff
@@ -14,7 +14,7 @@ public function index()
     {
```

Comment:
(This comment doesn't have to do with this line of code. I'm just creating a "conversation" that can be "resolved".)

I'm trying to run this. I ran the following:

```bash
sudo su;
apt update && apt install -y zip unzip git npm;
cd /app;
php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');";
php -r "if (hash_file('sha384', 'composer-setup.php') === 'dac665fdc30fdd8ec78b38b9800061b4150413ff2e3b6f88543c636f7cd84f6db9189d43a81e5503cda447da73c7e5b6') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;";
php composer-setup.php;
php -r "unlink('composer-setup.php');";
mv composer.phar /usr/bin/composer;
composer install;
sed 's/^APP_URL.*/APP_URL=http:\/\/localhost:8000/' .env.example > .env;
php artisan key:generate;
php artisan migrate;
npm i;
php artisan serve --host 0.0.0.0;
```
When I visit http://localhost:8000, I get

`Vite manifest not found at: /app/public/build/manifest.json`

Reply:
Oh, I was able to get around that by running `npm run dev` as you instructed!